docs/2.guide/1.concepts/1.auto-imports.md

```diff
@@ -140,3 +140,26 @@ export default defineNuxtConfig({
   }
 })
 ```
+
+## Auto-import from third-party packages
+
+Nuxt also allow auto-importing from third-party packages by configuring `imports:sources` lifecycle hook.
+
+::alert
+If you are using the Nuxt module for that package, it is likely that the module has already configured auto-imports for that package.
+::
+
+For example, You can enable auto-import of `useI18n` composable from `vue-i18n` package like below:
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  hooks: {
+    'imports:sources': sources => {
+      sources.push({
+        from: 'vue-i18n',
+        imports: ['useI18n'],
+      })
+    },
+  },
+})
```

Comment:
This might be a bit more straightforward in this case.

```suggestion
export default defineNuxtConfig({
  imports: {
    presets: [
      {
        from: 'vue-i18n',
        imports: ['useI18n']
      }
    ]
  }
})
```